internal/cmd/commands/dev/dev.go

```diff
@@ -335,6 +337,18 @@ func (c *Command) Flags() *base.FlagSets {
 		Usage:  "If set, the original \"ephemeral\" method of worker auth will be used to connect the initial dev worker to the controller.",
 	})
 
+	f.StringVar(&base.StringVar{
+		Name:   "worker-storage-dir",
```

Comment:
There's an inconsistency between the flag name -- which doesn't have "auth" in it -- and the variable names/usage strings, which indicate it's for auth storage. I'm of two minds about the right way to proceed. On the one hand we could define a single top level storage dir using these flags and have auth and perhaps other future purposes be under this one directory. On the other hand for future purposes we may want to allow defining storage in different locations.

My inclination is to say that we should change the flag name to `worker-auth-storage-dir`/`worker-auth-storage-skip-cleanup`.